Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM elifesciences/php_cli
+
+USER root
+RUN mkdir /srv/api-dummy && chown elife:elife /srv/api-dummy
+
+USER elife
+WORKDIR /srv/api-dummy
+COPY composer.json composer.lock /srv/api-dummy/
+RUN composer install
```

Comment:
If this isn't meant for dev/testing, `--classmap-authoritative --no-dev` would help speed it up.

Reply:
we can probably start with that now and later parameterize it for dev/testing vs. deployment

Reply:
And exclude `test`?